en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md

```diff
@@ -617,6 +612,9 @@ Injector.transform(
 );
 ```
 
+> [!WARNING]
+> It is critical that you end series of mutation calls with `getState()`.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.